outage/2024-10-09-ota-lite.md

```diff
@@ -0,0 +1,57 @@
+## OTA Lite Outage
+
+OTA-lite was unavailble from 20:13UTC to 20:35UTC. During this time things
+like publishing TUF Targets and managing devices (from user facing APIs)
+was not possible. The device-gateway remained up during this outage.
+
+### What Happened
+
+On the previous day, 2024-10-08, we performed a task where we periodically
+rotate secure keys and tokens used by our backend. After rotating keys, we
+restarted each container affected by the key rotation to make sure they
+continuted to function properly. We also deleted some old service accounts
+and tokens no longer used by our system.
+
+We went to bed and slept comfortably.
+
+Later the next day OTA lite would not start up correctly after we tried
+to deploy a very minor change to it. We tried to rollback the change only
+to discover it couldn't run either.
```

Comment:
I would order these events differently, to be more precise about their consequentiality:
```suggestion
Later the next day we tried to deploy a minor change to the API, but it was failing to start up.
We tried to rollback the change only to discover it couldn't run either.
```

Maybe, it is worth saying `api.foundries.io` instead of `the API` above, it's up to you.